### DISCUSSION TITLE
Sign in request says it has no request.

### DISCUSSION THREAD

**Discussion by Matt-17** [community/community#146727] [Other Feature Feedback, Questions, & Ideas]
Usually I'm logged in on my PC, but sometimes it's not. Ok, So I want to log in and enter my credentials. Then it's telling me, I need my Github Mobile app to authorize my access. I get a push notification and when I click it, it says there is no authorization request information. Then I cannot do anything. After a while I can retry on PC to authorize but the same happens. 

After a while nwo I tried again and it worked as it should. 

Edit: Updated this text on PC after I finally could log in. Also thanks for choosing the correct category, it was not obvoius which one to pick on this list when in a hurry.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Matt-17 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
